src/components/charts/SummaryBarChart.jsx

```diff
@@ -0,0 +1,139 @@
+  color: '#3283c8',
+};
+
+const calculateBarChartData = (data, showPercentage, percentageFixedPoint) => {
+  if (showPercentage) {
+    let sum = 0;
+    data.forEach((d) => { sum += d.value; });
+    let percentLeft = 100;
+    return data.map((entry, index, array) => {
+      let percentage;
+      if (index < array.length - 1) {
+        percentage = (entry.value * 100) / sum;
+      } else {
+        percentage = percentLeft;
+      }
+      percentage = Number(Number.parseFloat(percentage).toFixed(percentageFixedPoint));
+      percentLeft -= percentage;
+      return Object.assign({ percentage }, entry);
+    });
+  }
+  return data;
+};
+
+class SummaryBarChart extends React.Component {
+  render() {
+    const getCategoryColor = (index) => {
+      // map index to (1-9)
+      const i = (index % 9) + 1;
+      return this.props.localTheme[`barGraph.bar${i}Color`];
+    };
+    const monoFillColor = this.props.monoColor ? this.props.color : undefined;
+    const barChartHeight = (this.props.data.length * this.props.barSize)
+      + ((this.props.data.length + 1) * this.props.barGap) + 2;
+    const barChartData = calculateBarChartData(
+      this.props.data,
+      this.props.showPercentage,
+      this.props.percentageFixedPoint,
+    );
+    const dataKey = this.props.showPercentage ? 'percentage' : 'value';
+    const toPercentageFormatter = v => (this.props.showPercentage ? `${v}%` : v);
+    return (
+      <BarChartWrapper>
+        <BarChartTitle className="h4-typo">
+          {this.props.title}
+        </BarChartTitle>
+        <ResponsiveContainer width="100%" height={barChartHeight}>
+          <BarChart
+            layout={this.props.vertical ? 'vertical' : 'horizontal'}
+            data={barChartData}
+            barCategoryGap={this.props.barGap}
+            barSize={this.props.barSize}
+            margin={{ top: 4, right: 35, left: 15 }}
+          >
+            <Tooltip formatter={toPercentageFormatter} />
+            <XAxis axisLine={false} tickLine={false} type="number" hide />
+            <YAxis axisLine={false} tickLine={false} dataKey="name" type="category" style={yAxisStyle} interval={0} />
+            <Bar dataKey={dataKey}>
+              {
+                barChartData.map((entry, index) => (
+                  <Cell key={'bar'.concat(index)} fill={monoFillColor || getCategoryColor(index)} />
+                ))
+              }
+              <LabelList dataKey={dataKey} position="right" offset={8} style={labelValueStyle} formatter={toPercentageFormatter} />
+            </Bar>
+          </BarChart>
+        </ResponsiveContainer>
+      </BarChartWrapper>
+    );
+  }
+}
+
+const ChartDataShape = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  value: PropTypes.number.isRequired,
+});
+
+SummaryBarChart.propTypes = {
+  title: PropTypes.string.isRequired,
+  data: PropTypes.arrayOf(ChartDataShape).isRequired,
+  localTheme: PropTypes.object.isRequired,
+  vertical: PropTypes.bool,
```

Comment:
I'm not sure these `vertical` and `monoColor` do anything - when I remove them in Storybook nothing changes

Reply:
There is something wrong with "vertical", I am still checking...I think "monoColor" should work 😛